### DISCUSSION TITLE
Getting wrong Battery Voltage on ELRS ER5A

### DISCUSSION THREAD

**Discussion by miles2912** [ExpressLRS/ExpressLRS#2039] [General]
Pack tested on a Frsky RX6R - and Radiomaster TX16s.  Telemetry shows and volt meter confirms 5.5v.  Plug in the ER5A and it shows 4.0v then over a minute drops to 2.9v.   The pack is a 4.8v 1000mah flat Receiver pack.  TX16s is on EDGE 2.8 and ELRS is 3.1.2 on RX and RadioMaster Ranger Micro TX.

**Comment by fraze1974** [comment]
I had the same with the ER5C which from what I can tell is the same RX inside the ER5A just with different header pin orientations. I emailed Radiomaster and this was their response:
Hi Fraze,
Sorry, RxBat telemetry is not supported on the ER5C. 
Kind regards,
James
